packages/prisma/migrations/20240722105401_add_is_admin_api_enabled_organization_setting/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "OrganizationSettings" ADD COLUMN     "isAdminAPIEnabled" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
Should we add an update to set all existing records to true so no one loses functionality upon deployment?

Reply:
Yeah I think that would make sense